### PR TITLE
Minor fixes for EOS 5.0

### PR DESCRIPTION
**Pull request by jprvita**
These are mostly whitespace fixes, but there are a couple real fixes in there too (preposition and typos).

### CODE REVIEW

**Comment by jprvita** [comment]
The final commit fixes the indentation of Makefile.am -- I recall Makefiles being very particular about indentation, so maybe some figures were not being included before that, but I have not checked (or built this at all).

File: gnome-help/C/backup-what.page [hunk 1]
```diff
@@ -54,8 +54,8 @@
  <item>
   <title>System settings</title>
    <p>Most people never change the system settings that are created during
-   installation. If you do customize your system settings for some reason 
-   then you may wish to back up these settings.</p>
+   installation. If you do customize your system settings for some reason
```
Comment (cassidyjames) [inline gnome-help/C/backup-what.page:57]: I think this reads awkwardly with no comma, but I agree the "then" is not needed.

```suggestion
   installation. If you do customize your system settings for some reason,
```

**Review by cassidyjames — APPROVED** [review]
I think it's fine, but stylistically I'd add a comma in the spot I mentioned. 🤷

File: gnome-help/C/backup-what.page (hunk 1 above)
Comment (jprvita) [reply gnome-help/C/backup-what.page:57]: Sounds good! I was not sure if that called for a comma or not, so it is great to have an extra opinion :)

**Pull request merged**